lib/image_optim/worker/gifsicle.rb

```diff
@@ -32,6 +36,26 @@ def self.init(image_optim, options = {})
         OptionHelpers.limit_with_range(v.to_i, 1..3)
       end
 
+      LOSSY_OPTION =
+      option(:lossy, 0, 'lossy compression level (requires giflossy). `\
+        `Value type is signed integer. Use low positive values for better `\
+        `quality or low negative values for extremly bad quality. `\
+        `Example values:'\
+          '`0` - no lossy compression, '\
+          '`20` - low level compression (low noise), '\
+          '`1000` - high level of compression (high noise), '\
+          '`-1` - max level of compression (max noise)') do |v, opt_def|
+        if v.to_i == opt_def.default
```

Comment:
This path can be removed as it will be handled by `allow_lossy` path

Reply:
If it's removed, the warning message will be displayed on default config: allow_lossy: false, lossy: 0.